packages/repository-modernizer/src/restructure-config.js

```diff
@@ -338,5 +340,152 @@ async function deleteEmptyConfigDir(oldConfigFilePath) {
         oldConfigFileParent = path.dirname(oldConfigFileParent);
     }
 }
+/**
+ *
+ * @param String newConfigFilePath the new config file path in ui.config package
+ *
+ * format config in xml ,cfg,config format to cfg.json
+ */
+async function formatConfig(newConfigFilePath) {
```

Comment:
The param name should be `osgiConfigFilePath`, since this method's only purpose is reformatting OSGi config files (i.e. it should not matter/care whether the path being provided is old or new).